eng/platform-matrix.yml

```diff
@@ -79,6 +79,27 @@ jobs:
         - Alpine.38.Amd64
     ${{ insert }}: ${{ parameters.jobParameters }}
 
+# Linux musl arm64
+
+- template: ${{ parameters.jobTemplate }}
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    archType: arm64
+    osGroup: Linux
+    osIdentifier: Linux_musl
+    containerName: musl_arm64_build_image
+    helixQueues:
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        # TODO: there are no Alpine arm64 queues https://github.com/dotnet/core-eng/issues/5206
```

Comment:
I think this is not true. According to 
https://helix.dot.net/api/2018-03-14/info/queues there are 
**Alpine.3.Arm64** and **Alpine.3.Arm64.Open**

Can you please add them here?

Reply:
@ulisesh what docker image is this queue running.

Reply:
@echesakovMSFT after some more thought, getting clean runs I believe should be out of scope for this change. It currently blocks up-stack components from building. I will enable testing in a separate PR unless there is strong objection.

Reply:
I am fine with merging this as is and opening another PR to target testing side.